.github/workflows/ci.yml

```diff
@@ -194,6 +194,25 @@ jobs:
             project_tags_cmake_options: "-DROBOTOLOGY_PROJECT_TAGS=Custom -DROBOTOLOGY_PROJECT_TAGS_CUSTOM_FILE=${GITHUB_WORKSPACE}/releases/2021.02.yaml"
     steps:
     - uses: actions/checkout@master
+    
+    - name: Move robotology-superbuild in C under Windows
+      if: contains(matrix.os, 'windows')
+      shell: bash
+      run: |
+        ls /d/
+        cp -r ${GITHUB_WORKSPACE} /c/
```

Comment:
What about `mv` instead of `cp` ?

Reply:
I tried, and it was complaining that the directory was used (probably there is some process monitoring its content), so that's the reason why I went for copying it.

Reply:
Ok nevermind 👍🏻